ui/src/kapacitor/components/config/AlertaConfig.tsx

```diff
@@ -1,39 +1,49 @@
-import React, {Component} from 'react'
-import PropTypes from 'prop-types'
+import React, {PureComponent} from 'react'
 
 import RedactedInput from './RedactedInput'
+import {Input} from 'src/types/kapacitor'
 
-class AlertaConfig extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      testEnabled: this.props.enabled,
-    }
+interface Properties {
+  environment: string
+  origin: string
+  token: string
```

Comment:
The actual value being passed in is a string and that is what gets saved into the DB. But because that field is redacted, it is passed as a boolean in config to indicate whether there is a value or not.